modules/statics/src/coins.ts

```diff
@@ -399,14 +399,16 @@ export const coins = CoinMap.fromCoins([
     'avaxp',
     'Avalanche P-Chain',
     Networks.main.avalancheP,
-    UnderlyingAsset.AVAXP
+    UnderlyingAsset.AVAXP,
+    [CoinFeature.STAKING]
```

Comment:
Please add this to `AVAXPCoin.DEFAULT_FEATURES` instead (See failing test) or use spread syntax[...]